Add share buttons at the top of each blog post

josecelano opened the pull request:
This PR fixes @grmbyrn's PR: https://github.com/torrust/torrust-website/pull/129

Commit:
Add share buttons at the top of each blog post

grmbyrn committed:
Update ShareButton.svelte

* Encoded parameters in URL.
* Added title to tweet message.

grmbyrn committed:
Update ShareButton.svelte

Encoded body and title

grmbyrn committed:
Update ShareButton.svelte

Encoded all of body into one encodeURIComponent.

josecelano committed:
feat: add share buttoms at the top of each blog post

grmbyrn reviewed (commented):
Hi @josecelano, thanks for your feedback and for addressing the issues in the recent pull request.

I've reviewed the changes in the new PR, and it looks like the issues regarding the missing injected title and the URL encoding have been successfully resolved. I agree with the changes, and they seem to address the concerns raised. All four means of sharing the blog post continue to work for me locally.

If you have any additional instructions or if there's anything else I should look into, please let me know. Otherwise, I'm ready to proceed with the changes and merge the PR.

josecelano commented:
> Hi @josecelano, thanks for your feedback and for addressing the issues in the recent pull request.
> 
> I've reviewed the changes in the new PR, and it looks like the issues regarding the missing injected title and the URL encoding have been successfully resolved. I agree with the changes, and they seem to address the concerns raised. All four means of sharing the blog post continue to work for me locally.
> 
> If you have any additional instructions or if there's anything else I should look into, please let me know. Otherwise, I'm ready to proceed with the changes and merge the PR.

Hi @grmbyrn, It's OK
